### ISSUE TITLE
Repository name isn't helpful when forking

### ISSUE THREAD

**Issue by freakboy3742** [pyvideo/data#707]
The name of the pyvideo/data repository is exceptionally generic - while it makes sense in the context of the pyvideo account, if a potential contributor forks the repository into their own Github account in order to make a contribution, they end up with a "<username>/data" repository that isn't especially meaningful:

<img width="1008" alt="screen shot 2018-12-03 at 8 32 01 am" src="https://user-images.githubusercontent.com/37345/49347103-f72a1f80-f6d5-11e8-8ecb-b05fc4e1d18c.png">

While an end-user *can* rename the repository once they've forked it, it would be better as a user-experience if this step wasn't required.

Suggested name: `pyvideo-data`.

**Comment by redapple** [comment]
Thank @freakboy3742 for bringing this up. +1
I've been cloning to a custom `pyvideo-data/` folder myself

**Comment by Daniel-at-github** [comment]
It's a good point.
The repo name it's harcoded in a lot of places:

- Git submodules
- Internal scripts
- Travis
- ...

If we rename the github repo and add a symlink `pyvideo-data` => `data`... will travis and the submodules work?

**Comment by freakboy3742** [comment]
Yes - I imagine there will be plenty of changes that need to be made in the pipeline, and there will be some friction for any users that have an `upstream` remote in their own checkouts. The question (to me, anyway) is whether the temporary friction associated with making the change exceeds the long term benefit to contributors of having a "better" repository name. I'll leave that up to project maintainers to decide if that's a worthwhile cost.
